src/Microsoft.Health.Dicom.Api/Features/Filters/AcceptContentFilterAttribute.cs

```diff
@@ -13,22 +13,17 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.Filters;
 using Microsoft.Extensions.Primitives;
-using Microsoft.Health.Dicom.Core.Web;
 using Microsoft.Net.Http.Headers;
 
 namespace Microsoft.Health.Dicom.Api.Features.Filters;
 
 public sealed class AcceptContentFilterAttribute : ActionFilterAttribute
 {
     private const int NotAcceptableResponseCode = (int)HttpStatusCode.NotAcceptable;
-    private const string TypeParameter = "type";
-
-    private readonly bool _allowSingle;
-    private readonly bool _allowMultiple;
 
     private readonly HashSet<MediaTypeHeaderValue> _mediaTypes;
 
-    public AcceptContentFilterAttribute(string[] mediaTypes, bool allowSingle, bool allowMultiple)
+    public AcceptContentFilterAttribute(string[] mediaTypes)
```

Comment:
Can we confirm this change also works down stream? There are places like dicom-server\src\Microsoft.Health.Dicom.Api\Extensions\MediaTypeHeaderValueExtensions.cs, where the acceptHeader gets converted to internal objects and passed into service, is the type with and without quotes passed in as same value downstream?

Reply:
The service currently uses two code paths to handle media type content negotiation - `AcceptContentFilterAttribute` is used only for single part media types like `application/dicom`, and the `GetAcceptHeaders` extension method is only used on controller actions that use the `Produces` attribute for `multipart/related` media types, and then parses to find the media type from the `type` parameter. 

Once we realized that the `AcceptContentFilterAttribute` doesn't handle `multipart/related`, we decided to explicitly handle only single part media types there, and consider a future refactor to handle multipart media type content negotiation at the API layer as well.